packages/javascript/bh-shared-ui/package.json

```diff
@@ -25,7 +25,6 @@
         "@fortawesome/react-fontawesome": "^0.2.0",
         "@reduxjs/toolkit": "^1.9.5",
         "clsx": "^1.1.1",
-        "dompurify": "^2.3.0",
```

Comment:
Will you please make these same changes in the package.json under cmd/ui?

Reply:
Actually I was looking at the wrong one. The changes here look good imo. Thanks for the clean up!